### PR TITLE
Add docs for admin group without GitHub app.

### PR DESCRIPTION
**Pull request by Xantier**

### CODE REVIEW

**Comment by dtuite** [comment]
https://deploy-preview-684--roadie.netlify.app/docs/getting-started/create-admin-group/

File: content/docs/getting-started/create-admin-group/index.md [hunk 1]
```diff
@@ -12,11 +12,17 @@ concept of admins to enable this.
...
 
-You will need to have [installed the Roadie GitHub app](/docs/gettting-started/install-github-app/) before proceeding with this step.
+For automatic Admin group management you will need to have [installed the Roadie GitHub app](/docs/gettting-started/install-github-app/) before proceeding with this step.
+
+For manual management you should be aware of the correct usernames of the users you are assigning to admin group
 
 ## Create an admin group and user
 
-Roadie admins have special powers such as the ability to add and remove Backstage plugins. We use GitHub Teams to designate admins.
+Roadie admins have special powers such as the ability to add and remove Backstage plugins.
```
Comment (dtuite) [inline content/docs/getting-started/create-admin-group/index.md:21]: The special powers of admins were already mentioned in the intro, so I wonder if it makes sense to cut this paragraph and the heading "Create an admin group and user" and turn the h3's below into h2's instead. Then they'll be visible in the right sidebar, which might be useful.

File: content/docs/getting-started/create-admin-group/index.md [hunk 2]
```diff
@@ -12,11 +12,17 @@ concept of admins to enable this.
...
+For manual management you should be aware of the correct usernames of the users you are assigning to admin group
 
 ## Create an admin group and user
 
-Roadie admins have special powers such as the ability to add and remove Backstage plugins. We use GitHub Teams to designate admins.
+Roadie admins have special powers such as the ability to add and remove Backstage plugins. 
+
+### Automatic discovery with GitHub teams
+
+To automatically designate admin you can use GitHub Teams which will be automatically discovered if you have installed the Roadie GitHub application.
```
Comment (dtuite) [inline content/docs/getting-started/create-admin-group/index.md:25]:
```suggestion
To automatically designate admins you can use GitHub Teams which will be automatically discovered if you have installed the Roadie GitHub application.
```

File: content/docs/getting-started/create-admin-group/index.md [hunk 3]
```diff
@@ -33,3 +39,49 @@ Roadie admins have special powers such as the ability to add and remove Backstag
 ![A link that says "Log in to GitHub"](./sidebar-log-into-github.png)
 
 ⚠️  &nbsp;It can take some time for Roadie to refresh the list of teams from GitHub teams. If you do not see admin functions immediately, please wait a few minutes and try again.
+
+
+### Manual admin group configuration with group entities
```
Comment (dtuite) [inline content/docs/getting-started/create-admin-group/index.md:44]: I think I'd add "(advanced alternative)" or something after this. To indicate that it's an optional step.

File: content/docs/getting-started/create-admin-group/index.md [hunk 4]
```diff
@@ -33,3 +39,49 @@ Roadie admins have special powers such as the ability to add and remove Backstag
...
+
+In case you don't want to install the GitHub app or don't want to manage teams via you can also manually add groups into your Roadie instance.
+
+1. Make a note of the user entity that is assigned to you on the account page of users. You can find this information from `https://<your-tenant>.roadie.so/Account`.
+   * In case you have not logged into GitHub, this is usually the first part of your email address before the `@` symbol.
+
+2. Create a `catalog-info.yaml` entity definition for `roadie-backstage-admin` group
```
Comment (dtuite) [inline content/docs/getting-started/create-admin-group/index.md:51]: Where do I create this catalog-info.yaml? 

I wonder if we can make an OS repo for people to fork into their orgs to make this even easier. That can be a separate ticket of course.

Comment (Xantier) [reply content/docs/getting-started/create-admin-group/index.md:51]: At this stage I would expect people to be familiar of catalog-info files tbh. An alternative onboarding repo for non-GH is probably is a good idea.

File: content/docs/getting-started/create-admin-group/index.md (hunk 1 above)
Comment (Xantier) [reply content/docs/getting-started/create-admin-group/index.md]: Yup, removed the h2 and modified those below h3s to be more descriptive h2s instead.

File: content/docs/getting-started/create-admin-group/index.md [hunk 5]
```diff
@@ -12,11 +12,14 @@ concept of admins to enable this.
 
 ## Prerequisites
 
-You will need to have [installed the Roadie GitHub app](/docs/gettting-started/install-github-app/) before proceeding with this step.
+For automatic Admin group management you will need to have [installed the Roadie GitHub app](/docs/gettting-started/install-github-app/) before proceeding with this step.
```
Comment (Xantier) [inline content/docs/getting-started/create-admin-group/index.md:15]: There's three `t`s on this URL.

Comment (Xantier) [inline content/docs/getting-started/create-admin-group/index.md:15]:
```suggestion
For automatic Admin group management you will need to have [installed the Roadie GitHub app](/docs/getting-started/install-github-app/) before proceeding with this step.
```

**Review by dtuite — APPROVED** [review]

**Pull request merged**